examples/sushi/macros/utils.py

```diff
@@ -1,9 +1,20 @@
+from sqlglot import exp
+
 from sqlmesh import macro
 
 
 @macro()
-def add_one(evaluator, column):
-    return f"{column} + 1"
+def add_one(evaluator, column: int):
```

Comment:
Just a heads up that I don't think that this is how this meant to be used. I think it's suppose to return the SQL `Add` expression, not perform the operation in place.